src/app/app.js

```diff
@@ -1,32 +1,58 @@
 import express from 'express';
+import cookieSession from 'cookie-session';
 import helmet from 'helmet';
 import pinoMiddleware from 'express-pino-logger';
 import compression from 'compression';
 import staticGzip from 'express-static-gzip';
+import fetch from 'node-fetch';
```

Comment:
`node-fetch` won't play nicely with webpack when it does a production build 😭  

... it looks like this is only used in the "TMP" commit, so it's probably not something to worry about right now 

... IIRC [axios](https://github.com/axios/axios) is quite a popular promisey/http lib

Reply:
Yeah, it's part of our TMP commit just to prove a point. It should go and ideally be replaced by node-cfapi of some sort.